source/renderer/app/api/utils/localStorage.js

```diff
@@ -322,6 +326,30 @@ export default class LocalStorageApi {
   unsetAppUpdateCompleted = (): Promise<void> =>
     LocalStorageApi.unset(keys.APP_UPDATE_COMPLETED);
 
+  getAssetsLocalData = (): Promise<AssetLocalData> =>
+    LocalStorageApi.get(keys.ASSET_DATA, []);
+
+  unsetAssetsLocalData = (): Promise<void> =>
+    LocalStorageApi.unset(keys.ASSET_DATA);
+
+  getAssetLocalData = (fingerprint: string): Promise<AssetLocalData> =>
+    LocalStorageApi.get(keys.ASSET_DATA, {}, fingerprint);
```

Comment:
@daniloprates we should be fetching the data by `policyId+assetName` and not `fingerprint`.